cypress/support/commands.js

```diff
@@ -23,3 +25,96 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('fillForm', (userData) => {
+  // enter firstname:
+  cy.get('#firstName').type(userData.firstName);
+
+  // enter lastname:
+  cy.get('#lastName').type(userData.lastName);
+
+  // enter email:
+  cy.get('#userEmail').type(userData.email);
+
+  // select gender:
+  const userGender = userData.gender;
```

Comment:
deconstruct all user data from userData object before filling the form

Reply:
fixed